File: Iniciante/URI 1010.c
#include <stdio.h>
 
int main() { 
    int cod1, cod2, n1, n2;
    float valor1, valor2, total;

    scanf("%d %d %f", &cod1, &n1, &valor1);
    scanf("%d %d %f", &cod2, &n2, &valor2);
    total = (n1 * valor1) + (n2 * valor2);
    printf("VALOR A PAGAR: R$ %.2f\n", total);

    return 0;
}

File: Iniciante/URI 1010.ml
let cod1, n1, valor1, cod2, n2, valor2 = Scanf.scanf "%i %f %f\n%i %f %f" (fun a b c d e f -> a, b, c, d, e, f);;
let total = (n1 *. valor1) +. (n2 *. valor2);;
Printf.printf "VALOR A PAGAR: R$ %.2f\n" (total);